dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/EndpointTests/SignIn/TestBase.CommonTests.cs

```diff
@@ -167,12 +167,12 @@ public async Task MissingAuthenticationState_ReturnsBadRequest(HttpMethod method
     }
 
     public async Task ValidRequest_RendersContent(
+        Func<Configure, Func<AuthenticationState, Task>> configureAuthenticationHelper,
```

Comment:
I flipped the parameter order here so this method is consistent with the others.